project_name/tests/test_sign.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
There's one more thing failing in the "pre-commit" check.

Did you use `pre-commit` when committing your code? It automatically sorts imports and tells you about style issues before it gets to GitHub.

The output here:

```diff
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing /home/runner/work/pcds-ci-test-repo-python/pcds-ci-test-repo-python/project_name/tests/test_sign.py

pre-commit hook(s) made changes.
If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
To run `pre-commit` as part of git workflow, use `pre-commit install`.
All changes made by hooks:
diff --git a/project_name/tests/test_sign.py b/project_name/tests/test_sign.py
index 6fc37e4..14c7ffc 100644
--- a/project_name/tests/test_sign.py
+++ b/project_name/tests/test_sign.py
@@ -1,5 +1,5 @@
-import pytest
 import numpy as np
+import pytest
```

Tells you that `isort` would have put `numpy` before `pytest`, because they are sorted alphabetically by convention.

To fix it, you can do as it says above:
```
$ pre-commit install
$ pre-commit run --all-files
```

Or manually fix based on what it says, of course. I'm lazy and prefer to have pre-commit take care of it for me, though!